### ISSUE TITLE
command not found: `salloc`

### ISSUE THREAD

**Issue by vict0rsch** [mila-iqia/milatools#3]
```
$ mila code ~/ccai/github --alloc --gres=gpu:1 --mem=16G --partition=main
(local) $ ssh mila -fNMS /Users/victor/.ssh/sockets/milatools.mila
(mila) $ salloc --gres=gpu:1 --mem=16G --partition=main
# zsh:1: command not found: salloc
ERROR: Could not find the node name for the allocation
```

I think a way to solve this is by adding `. /etc/profile` at login before calling `salloc` but I don't really know why or what it does, I remember it as a quick fix

**Comment by breuleux** [comment]
Yeah, I was wondering how it'd interact with other shells than `bash`.

I want to try to keep the commands simple, so I'm tempted to just hardcode the `/opt/slurm/bin/salloc` path it should be at, although that may not be very robust (but it would be easy enough to update).

**Comment by vict0rsch** [comment]
Ok, let me know how I can help support `zsh` :)

**Comment by breuleux** [comment]
I made it use `bash -c salloc` instead of `salloc`, which should solve the issue in most cases.

For the record, adding `. /etc/profile` in `.zshenv` should fix the issue for `zsh` users, since that file is sourced even when the shell is used to execute a single command.
